### DISCUSSION TITLE
Global configuration - exclusions

### DISCUSSION THREAD

**Discussion by JohnDBallentineIII** [trimble-oss/dba-dash#183] [Q&A] [answered]
I may be missing this, so I'm not putting it in as an issue.  As one example - in my environment, we don't do 'dbcc checkdb' on tempdbs.  Without getting into an argument about whether or not we should be running this, is there any way to tell DBADash 'ignore this' as a global configuration instead of going into each server individually?  

Otherwise, I really like this tool!  As others have commented, came over here from Brent Ozar's blog post, and I've not regretted it - I've wanted something like this for years, and always gotten push-back because of funding.  Since my time is 'free', this works for my company...

Thanks!
John Ballentine

**Comment by DavidWiseman** [comment, marked as answer]
Yes you can do this. 😊

- Select the "Checks" node in the tree
- Select the "Last Good CheckDB" tab
- Select "Configure Root Thresholds" from the menu as shown below:

![image](https://user-images.githubusercontent.com/33157668/166474271-2b40dd7e-e839-4dd7-adb9-4e7f1618aa9c.png)

- Enter a comma-separated list of databases you want to exclude from all servers (or just enter tempdb)

![image](https://user-images.githubusercontent.com/33157668/166474509-52ada07a-f099-470a-8747-99c9dd7be99e.png)

You can set thresholds at the root level, instance-level, or customize them for individual databases.  At the root level you can specify a list of database names you want to exclude from all servers which makes it easy to exclude tempdb or other databases without having to do it individually on each server.

Glad you like the tool. Hope this helps

**Reply by JohnDBallentineIII** [reply]
Perfect, thanks!  It took me a bit to follow instructions (argh)...  I found that field exactly where you said it was, and now I no longer have a huge stack of red.  Just smaller, scattered ones...

I appreciate the work you've put in to this tool, and my manager was impressed by it too so it's a win all around.

**Reply by DavidWiseman** [reply]
Great.  It's good to get exclusions in place where you need them then you can focus on real issues when they are highlighted.
